platform/core/src/services/MeasurementService/MeasurementService.js

```diff
@@ -556,7 +563,7 @@ class MeasurementService {
    *
    * @param {MeasurementSource} source Measurement source instance
    * @param {string} definition The source definition
-   * @param {string} measurement The measurement serice measurement
+   * @param {MeasurementSource} measurement The measurement service measurement
```

Comment:
This is not a MeasurementSource, it's an Object, isn't it?

Reply:
Good catch, updated, thanks